HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -0,0 +1,1125 @@
+      <sch:assert test='count(h:NumberofConditionedFloors) = 1'>Expected 1 element(s) for xpath: NumberofConditionedFloors</sch:assert>
+      <sch:assert test='count(h:NumberofConditionedFloorsAboveGrade) = 1'>Expected 1 element(s) for xpath: NumberofConditionedFloorsAboveGrade</sch:assert>
+      <sch:assert test='count(h:NumberofBedrooms) = 1'>Expected 1 element(s) for xpath: NumberofBedrooms</sch:assert>
+      <sch:assert test='count(h:NumberofBathrooms) &lt;= 1'>Expected 0 or 1 element(s) for xpath: NumberofBathrooms</sch:assert>
+      <sch:assert test='count(h:ConditionedFloorArea) = 1'>Expected 1 element(s) for xpath: ConditionedFloorArea</sch:assert>
+      <sch:assert test='not(count(h:ConditionedBuildingVolume) = 0 and count(h:AverageCeilingHeight) = 0)'>Expected 1 or more element(s) for xpath: ConditionedBuildingVolume | AverageCeilingHeight</sch:assert>
+      <sch:assert test='count(h:extension/h:HasFlueOrChimney) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/HasFlueOrChimney</sch:assert>
+    </sch:rule>
+
+    <!-- [ClimateZone] -->
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:ClimateandRiskZones/h:ClimateZoneIECC'>
+      <sch:assert test='count(h:Year) = 1'>Expected 1 element(s) for xpath: Year</sch:assert>
+      <sch:assert test='count(h:ClimateZone[text()="1A" or text()="1B" or text()="1C" or text()="2A" or text()="2B" or text()="2C" or text()="3A" or text()="3B" or text()="3C" or text()="4A" or text()="4B" or text()="4C" or text()="5A" or text()="5B" or text()="5C" or text()="6A" or text()="6B" or text()="6C" or text()="7" or text()="8"]) = 1'>Expected 1 element(s) for xpath: ClimateZone[text()="1A" or text()="1B" or text()="1C" or text()="2A" or text()="2B" or text()="2C" or text()="3A" or text()="3B" or text()="3C" or text()="4A" or text()="4B" or text()="4C" or text()="5A" or text()="5B" or text()="5C" or text()="6A" or text()="6B" or text()="6C" or text()="7" or text()="8"]</sch:assert>
+    </sch:rule>
+
+    <!-- [WeatherStation] -->
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:ClimateandRiskZones/h:WeatherStation'>
+      <sch:assert test='count(h:SystemIdentifier) = 1'>Expected 1 element(s) for xpath: SystemIdentifier</sch:assert> <!-- Required by HPXML schema -->
+      <sch:assert test='count(h:Name) = 1'>Expected 1 element(s) for xpath: Name</sch:assert> <!-- Required by HPXML schema -->
+      <sch:assert test='(count(h:WMO) = 1 or count(h:extension/h:EPWFilePath) = 1)'>Expected 1 element(s) for xpath: WMO | extension/EPWFilePath</sch:assert> <!-- Reference weather/data.csv for the list of acceptable WMO station numbers -->
```

Comment:
Seems like sometimes this pattern is used, other times you use the following pattern. Would be nice to use the same pattern everywhere. 

```
<sch:assert test='(count(h:SystemType/h:Standard) + count(h:SystemType/h:Recirculation) = 1)'>
```

Reply:
This pattern was used for the first implementation. That'll be replaced with the new pattern to be consistent.